source/layouts/blog_article.erb

```diff
@@ -4,6 +4,15 @@
 <% content_for(:meta_keywords) { "unboxed consulting, news, technical, web, development, marketing, graphic design, Unboxed, Ruby on Rails, Blog, #{current_page.data.tags.join(', ')}" } %>
 <% add_atom_feed_to_head %>
 
+<!--Twitter card tags-->
+<% content_for(:head) do %>
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:site" content="@Ubxd">
+  <meta name="twitter:title" content="<%= current_page.data.title %>">
+  <meta name="twitter:description" content="<%= truncate(strip_tags(current_article.body), length: 150) %>">
+  <meta name="twitter:image" content="<%= current_page.data.weekly_roundup ? image_url("assets/images/blog/unboxed_roundup.jpg") : current_page.data.main_image %>">
```

Comment:
Another that has a main_image:

![screen shot 2016-10-26 at 15 03 48](https://cloud.githubusercontent.com/assets/885223/19729265/5a854776-9b8e-11e6-9aa3-4a7941c64dd2.png)